browser-test/src/admin/admin_program_translation.test.ts

```diff
@@ -492,4 +482,123 @@ test.describe('Admin can manage program translations', () => {
       )
     })
   })
+  test.describe('North Star translations tests', {tag: ['@northstar']}, () => {
+    test.beforeEach(async ({page}) => {
+      await enableFeatureFlag(page, 'north_star_applicant_ui')
+    })
+
+    test('Add translations for block name and description', async ({
```

Comment:
I moved this test down so I could nest it and my new test under the same `test.describe` and turn the north star flag on in the `beforeEach`.